### PR TITLE
tasks+omnibus: use a versioned SDS shared library.

### PR DESCRIPTION
**Pull request by remeh**
APR-94

### What does this PR do?

Use a pinned version of the SDS shared library.

### CODE REVIEW

File: omnibus/config/software/sds.rb [hunk 1]
```diff
@@ -1,14 +1,17 @@
...
 
 build do
     license "Apache-2.0"
     license_file "./LICENSE"
 
+    sds_version = "v0.1.0"
+
     # no Windows support for now.
     if linux_target? || osx_target?
+        command "git checkout #{sds_version}"
```
Comment (FlorentClarret) [inline omnibus/config/software/sds.rb:14]: Are you sure we actually need to explicitly checkout the version? I thought that was what `default_version` would do for you (but I can be wrong)

**Review by FlorentClarret — COMMENTED** [review]
Awesome thank you 🎉 ! I just left one question

File: omnibus/config/software/sds.rb (hunk 1 above)
Comment (chouquette) [inline omnibus/config/software/sds.rb:14]: I'm not 100% sure but think this isn't required. Omnibus should interpret the default version as a git ref and checkout automatically

Comment (remeh) [reply omnibus/config/software/sds.rb:14]: I'm giving it a shot!

Comment (remeh) [reply omnibus/config/software/sds.rb:14]: I'm giving it a shot!

Comment (remeh) [reply omnibus/config/software/sds.rb]: Seems to be working. Any chance you would know how to double-check this from the jobs worker or similar?

**Review by chouquette — APPROVED** [review]
LGTM!

**Review by FlorentClarret — APPROVED** [review]

**Pull request merged**